codegens/nodejs-axios/test/newman/newman.test.js

```diff
@@ -0,0 +1,27 @@
+var runNewmanTest = require('../../../../test/codegen/newman/newmanTestUtil').runNewmanTest,
+  convert = require('../../lib/index').convert;
+
+describe('Convert for different types of request', function () {
+  var options = {indentCount: 2, indentType: 'Space'},
+    testConfig = {
+      compileScript: null,
+      runScript: 'node run.js',
+      fileName: 'run.js',
+      headerSnippet: '/* eslint-disable */\n'
+    };
+
+  runNewmanTest(convert, options, testConfig);
+
+  describe('Convert for request incorporating ES6 features', function () {
```

Comment:
Since your variant does not have this option you can remove this test

Reply:
Same comment, add the support for ES6 syntax